homeassistant/components/climate/__init__.py

```diff
@@ -42,6 +42,7 @@
 STATE_COOL = 'cool'
 STATE_IDLE = 'idle'
 STATE_AUTO = 'auto'
+STATE_MANUAL = 'manual'
```

Comment:
I just remembered that @balloob thought that this should be a hold mode instead. See:
https://github.com/home-assistant/home-assistant/pull/11454#discussion_r171114275

Reply:
Does hold mode means that this temperature value should be hold? In the fritzbox implementation this will only work until the schedule configured in the fritzbox timesheet will switch again. e.g. 22:00 to  eco mode.

Reply:
Yeah. Ok, so then in the fritzbox case it doesn't sound as a hold mode feature since it doesn't override the schedule. So then this manual mode should still be operational mode? @balloob?